thanos/values.yaml

```diff
@@ -16,6 +16,7 @@ store:
   logLevel: debug
   securityContext: {}
   resources: {}
+  extraArgs: []
```

Comment:
Please remove these extraArgs because of #908

Reply:
Thanks, but I had a confusion with the #908 PR. I removed these changes from it - it should be merged here, as this is how it was tested.